### PR TITLE
Parallel and Extended workflows for quarkus backend app

### PR DESCRIPTION
**Pull request by tiagodolphine**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/swf-backend/src/service/router.ts [hunk 1]
```diff
@@ -161,13 +161,11 @@ function setupInternalRoutes(
...
     } = req;
     const swfData = req.body;
-    const svcResponse = await executeWithRetry(() =>
-      fetch(`${kogitoBaseUrl}:${kogitoPort}/${swfId}`, {
-        method: 'POST',
-        body: JSON.stringify(swfData),
-        headers: { 'content-type': 'application/json' },
-      }),
-    );
+    const svcResponse = await fetch(`${kogitoBaseUrl}:${kogitoPort}/${swfId}`, {
```
Comment (manstis) [inline plugins/swf-backend/src/service/router.ts:164]: Why did you remove the re-try?

File: plugins/swf-backend/src/service/router.ts [hunk 2]
```diff
@@ -254,6 +252,7 @@ function setupExternalRoutes(router: express.Router, discovery: DiscoveryApi) {
   });
 
   router.post('/actions/:actionId', async (req, res) => {
+    await delay(3000);
```
Comment (manstis) [inline plugins/swf-backend/src/service/router.ts]: Why the need for a 3s delay? That's quite long to wait for *every* action to be invoked.

File: plugins/swf-backend/src/service/router.ts (hunk 1 above)
Comment (tiagodolphine) [reply plugins/swf-backend/src/service/router.ts:164]: because if there is an error in the workflow execution it starts retying and this is not what we expect in this case, 
let's say there is a wrong parameter when starting the workflow (github repository URL) the workflow should be in error we can't start retrying.. I was getting this situation a lot when designing the workflow with error and compensation

File: plugins/swf-backend/src/service/router.ts (hunk 2 above)
Comment (tiagodolphine) [reply plugins/swf-backend/src/service/router.ts]: oh my mistake, I was simulating a delay to test timeout

**Review by manstis — APPROVED** [review]
